package.json

```diff
@@ -12,6 +12,7 @@
     "e2e": "ng e2e",
     "bundlesize": "bundlesize",
     "build:pkg": "ng build ngx-skeleton-loader --prod",
+    "build:pkg:trace": "node --trace-warnings ./node_modules/@angular/cli/bin/ng build ngx-skeleton-loader --prod",
```

Comment:
No need for this command here. Since it's a relevant issue I'm removing myself to get the issue quickly published